base/kubedriver.go

```diff
@@ -0,0 +1,68 @@
+package base
+
+import (
+	"errors"
+
+	// Import to initialize client auth plugins.
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
+
+	"github.com/iter8-tools/iter8/base/log"
+
+	"helm.sh/helm/v3/pkg/cli"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/rest"
+)
+
+// KubeDriver embeds Kube configuration, and
+// enables interaction with a Kubernetes cluster through Kube APIs
+type KubeDriver struct {
+	// EnvSettings provides generic Kubernetes options
+	*cli.EnvSettings
+	// RestConfig is REST configuration of a Kubernetes cluster
+	RestConfig *rest.Config
+	// DynamicClient enables unstructured interaction with a Kubernetes cluster
+	DynamicClient dynamic.Interface
+	// Namespace
+	Namespace *string
```

Comment:
We can get rid of the `Namespace` field.
***
*cli.EnvSettings already has Namespace method which [does the right thing](https://github.com/helm/helm/blob/cba3b1eed4209bf1a6d052bf63adf7fbadb6af7d/pkg/cli/environment.go#L176).

So, instead of `kd.Namespace`, we can do `kd.Namespace()` everywhere.

Reply:
done